viewing_party/party.py

```diff
@@ -1,23 +1,224 @@
 # ------------- WAVE 1 --------------------
 
+
+## Create a function that will create a movie function
+# with title, genre, and rating into a dictiionary
+# If there is no title, genre, or rating, return None
+# If any of the three variables is missing, return None
+
+from decimal import DivisionByZero
+from tests.test_constants import USER_DATA_2
+
+
 def create_movie(title, genre, rating):
-    pass
+
+    if not title or not genre or not rating:
+        return None
+    else: 

```

Comment:
There's nothing wrong with it, but we don't need to put this code in an else-block as long as it comes after the if-statement that checks for invalid data.